### ISSUE TITLE
WebSocket error when port forwarding to the web app

### ISSUE THREAD

**Issue by sahanDissanayake** [balderdashy/sails#2899]
Hi All,

I have my app running on port 1337 and I forward all my incoming requests to ubuntu EC2 port 80 to 1337
as described here
http://stackoverflow.com/questions/16573668/best-practices-when-running-node-js-with-port-80-ubuntu-linode?answertab=active#tab-top

But I get this error on the console
```javascript
WebSocket connection to 'ws://my-website.co/socket.io/1/websocket/vjtliBuQwvlwzRRqwTbz?__sails_io_sdk_version=0.10.0&__sails_io_sdk_platform=browser&__sails_io_sdk_language=javascript' failed: Connection closed before receiving a handshake response
```

But when I go to ``http://my-website:1337/ `` there are no errors

**Comment by sahanDissanayake** [comment]
Also seems like it is only on Google Chrome browser, Im using sails V0.10.5

**Comment by sahanDissanayake** [comment]
on the backend I get this error ``websocket connection invalid``

**Comment by sailsbot** [comment]
Thanks for posting, @sahanDissanayake.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by sailsbot** [comment]
Thanks for posting, @sahanDissanayake.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
